release-schema.json

```diff
@@ -120,8 +158,52 @@
             "boolean",
             "null"
           ]
+        },
+        "subcontracts": {
+          "title": "Subcontracts",
+          "description": "Information about the subcontracts that will be established.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Subcontract"
+          },
+          "uniqueItems": true,
+          "minItems": 1
+        }
+      },
+      "minProperties": 1
+    },
+    "Subcontract": {
+      "title": "Subcontract",
+      "description": "A relationship that the tenderers will establish with a third party to perform part of the contract on their behalf.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "Identifier",
+          "description": "The locally unique identifier for the subcontract.",
+          "type": [
+            "string"
+          ],
+          "minLength": 1
+        },
+        "subcontractor": {
+          "title": "Subcontractor",
+          "description": "The organization that will perform part of the contract on behalf of the tenderers.",
+          "$ref": "#/definitions/OrganizationReference"
+        },
+        "tenderers": {
+          "title": "Tenderers",
+          "description": "The tenderers to which the subcontractor will be subcontracted.",
```

Comment:
Some options:

* [Prime contractor](https://www.google.com/search?q=%22prime+contractor%22): 2.29m Google search results
* [Main contractor](https://www.google.com/search?q=%22main+contractor%22): 3.56m Google search results
* [General contractor](https://www.google.com/search?q="general+contractor"): 19m Google search results

According to the [Wikipedia page for 'General Contractor'](https://en.wikipedia.org/wiki/General_contractor), those terms are synonyms.

I remember 'prime contractor' coming up in discussions previously, but I think that 'main contractor' best communicates the nature of the role, although it has fewer search results than 'general contractor'.

Reply:
"general contractor" will get a lot of results because it's very common in construction.

I see "prime" often used in the Commonwealth. Both main and prime are used on europa.eu.

Let's go with "main".

Reply:
Done in https://github.com/open-contracting-extensions/ocds_subcontracting_extension/pull/6/commits/a6fdd61c175a2860372675bbf32e4963d035ecd0